Patch to disable nls (gettext) support for clamd. Taken from gentoo:
http://sources.gentoo.org/viewcvs.py/gentoo-x86/app-antivirus/clamav/files/clamav-0.94-nls.patch?view=log

--- clamav-milter/clamav-milter.c	Fri Nov 21 22:24:41 2008
+++ clamav-milter/clamav-milter.c	Mon Feb  2 09:30:14 2009
@@ -113,6 +113,7 @@
 
 #ifdef	C_LINUX
 #include <sys/sendfile.h>	/* FIXME: use sendfile on BSD not Linux */
+#ifdef ENABLE_NLS
 #include <libintl.h>
 #include <locale.h>
 
@@ -125,7 +126,8 @@
 #define	_(s)	s
 #define	N_(s)	s
 
-#endif
+#endif /* ENABLE_NLS */
+#endif /* C_LINUX */
 
 #ifdef	USE_SYSLOG
 #include <syslog.h>
@@ -774,7 +776,7 @@
 	else
 		progname = "clamav-milter";
 
-#ifdef	C_LINUX
+#ifdef	ENABLE_NLS
 	setlocale(LC_ALL, "");
 	bindtextdomain(progname, DATADIR"/clamav-milter/locale");
 	textdomain(progname);
--- configure.in	Wed Nov 26 12:15:08 2008
+++ configure.in	Mon Feb  2 09:30:14 2009
@@ -609,6 +609,14 @@
     fi
 fi
 
+AC_ARG_ENABLE([nls],
+               AC_HELP_STRING([--disable-nls], [disable NLS support]),
+               [want_nls=$enableval], [want_nls=yes]
+)
+if test $want_nls = yes; then
+       CPPFLAGS="$CPPFLAGS -DENABLE_NLS"
+fi
+
 AC_ARG_ENABLE([ipv6],
 [  --disable-ipv6          disable IPv6 support],
 want_ipv6=$enableval, want_ipv6="yes")
--- shared/getopt.c	Fri Nov 21 22:24:42 2008
+++ shared/getopt.c	Mon Feb  2 09:30:14 2009
@@ -82,7 +82,7 @@
 
 #ifndef _
 /* This is for other GNU distributions with internationalized messages.  */
-# if defined HAVE_LIBINTL_H || defined _LIBC
+# if (defined(HAVE_LIBINTL_H) || defined(_LIBC)) && defined(ENABLE_NLS)
 #  include <libintl.h>
 #  ifndef _
 #   define _(msgid)	gettext (msgid)
--- shared/output.c	Tue Nov 25 20:11:00 2008
+++ shared/output.c	Mon Feb  2 09:30:14 2009
@@ -61,7 +61,7 @@
 pthread_mutex_t logg_mutex = PTHREAD_MUTEX_INITIALIZER;
 #endif
 
-#ifdef  C_LINUX
+#if defined(C_LINUX) && defined(ENABLE_NLS)
 #include <libintl.h>
 #include <locale.h>
 
